contracts/Nexus.sol

```diff
@@ -13,7 +13,6 @@ pragma solidity ^0.8.26;
 // Learn more at https://biconomy.io. To report security issues, please contact us at: security@biconomy.io
 
 import { UUPSUpgradeable } from "solady/src/utils/UUPSUpgradeable.sol";
-import { EIP712 } from "solady/src/utils/EIP712.sol";
```

Comment:
why get rid of EIP712?

Reply:
it is in Module Manager now => inherited